examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
```

Comment:
<p>It seems like there are missing word here. I think everybody would agree.</p><p> </p><p>Also, maybe mention that we can also implement functions that are neither constant or balanced.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Fixed this paragraph!